src/sonic-frr/Makefile

```diff
@@ -10,7 +10,7 @@ STG_BRANCH = stg_temp.$(SUFFIX)
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Build the package
 	pushd ./frr
-	git checkout -b $(FRR_BRANCH) origin/$(FRR_BRANCH)
+	git checkout $(FRR_BRANCH) || git checkout -b $(FRR_BRANCH) origin/$(FRR_BRANCH)
```

Comment:
In what scenarios does this issue happen? I thought a normal rebuild should not trigger this since the branch should get removed after the build.

Reply:
Yeah, I notice that we clear branch at end. I think, on fresh Azure code, I notice FRR build failure first, and then i tried  to rebuild immediately, I notice 'branch exists' failure. So trying to addressed it. Seems, I should have tried to debug\resolve the first failure.  
Anyways, right now, if you do not see any side effect then we can put this fix, it helps ,while working on FRR patch in /patch/series folder. @shi-su

Reply:
Would this command `git checkout -b $(FRR_BRANCH) origin/$(FRR_BRANCH) || git checkout $(FRR_BRANCH)` work for this case? Though I currently could not think about any cases that cause side effects with your fix, I am a little unassured. I felt swapping the two commands may better align with the original behavior.

Reply:
Agree, let me change it and we can keep both commands.